gitbootcamp.rst

```diff
@@ -203,6 +203,33 @@ from the `core-workflow  <https://github.com/python/core-workflow>`_
 repository to backport the commit.
 
 
+.. _git_from_mercurial:
+
+Applying a Patch from Mercurial to Git
+--------------------------------------
+
+Scenario:
+
+- A Mercurial patch exists but there is no pull request for it.
+
+Solution:
+
+1. Download the patch locally.
+
+2. Create a new branch and switch to it.
+
+3. For Mac and Unix::
+
+   $ patch -p1 < /path/to/issueNNNN.patch
```

Comment:
I don't think there's much need to specify doing `git apply --check` first; if the patch won't apply cleanly, git will exit non-zero and not change anything.

And I'll note again, patches need not have been created with `hg diff --git` for `git apply` to be able to apply them.

I think `git apply` is the most foolproof way to make sure you get the full patch applied.  Try it from the HEAD of master first; if it fails, try updating to a revision from when the patch was created and apply again.  If it won't apply at that point, I'd say the patch is suspect and you're probably better off just reimplementing it rather than trying to get a patch tool to accept it.